daemon/src/rollover/maker.rs

```diff
@@ -120,28 +122,32 @@ impl Actor {
         } = msg;
         let order_id = propose.order_id;
 
-        if let Err(e) = self
+        let (from_event_id, from_complete_fee) = match self
             .executor
             .execute(order_id, |cfd| {
                 cfd.verify_counterparty_peer_id(&peer.into())?;
-                cfd.start_rollover()
+                cfd.start_rollover_maker(propose.from_commit_txid)
             })
             .await
         {
-            tracing::warn!(%order_id, "Rollover failed after handling taker proposal: {e:#}");
+            Ok(event_id) => event_id,
+            Err(e) => {
+                tracing::warn!(%order_id, "Rollover failed after handling taker proposal: {e:#}");
```

Comment:
Note by Daniel: I think we should change this into a `reject` scenario. The maker can send a `reject` back to the taker at this point. 

I did not change this yet because I am not sure about side-effects (I think they should be minimal, but still).

Reply:
see https://github.com/itchysats/itchysats/issues/2294